src/components/IOUConfirmationList.js

```diff
@@ -125,6 +116,8 @@ class IOUConfirmationList extends Component {
             value: props.hasMultipleParticipants ? CONST.IOU.IOU_TYPE.SPLIT : CONST.IOU.IOU_TYPE.REQUEST,
         }];
 
+        this.myPersonalDetails = _.findWhere(props.personalDetails, {isCurrentUser: true});
```

Comment:
NAB, making this a class instance property means it won't be updated in realtime. Seems fine to just re-access the details whenever we need them or create a `getCurrentUserPersonalDetails()` method.

Reply:
>What do you think about using "current user" as the subject instead of "my"?
Yeah, perhaps that is more clear. I'm down with that. 

>current user login should be available in session.email (if I am remembering correctly)
Yes, this is correct. When chatting with @tgolen about the approach, we were just thinking that having to subscribe every file that users the current user's personal details to the session key in Onyx would be overkill. But I like the idea of just creating a `getCurrentUserPersonalDetails()` method in the `PersonalDetails` lib. I'll do that.

Reply:
That approach sounds fine to me. So we'd be passing `props.personalDetails` to that method? Only other though is that it should go in a `PersonalDetailsUtils` (if that exists) as it's not an "action".